### PR TITLE
1.9 recovery

### PR DESCRIPTION
**Pull request by luxaritas**

### CODE REVIEW

**Comment by luxaritas** [comment]
I think that last commit also resolves #432

File: assets/Styles/styles.css [hunk 1]
```diff
@@ -44,3 +53,8 @@
         background-color: rgba(0, 16, 38, 1) !important;
     }
 }
+
+.eterna-input::placeholder {
+    /* If you update this, make sure to also change it in TextInputObject.ts */
```
Comment (everyday847) [inline assets/Styles/styles.css:58]: Let's file an issue to make these two colors linked explicitly? Long term.

File: src/eterna/mode/DesignBrowser/DesignBrowserMode.ts [hunk 2]
```diff
@@ -820,7 +820,7 @@ export default class DesignBrowserMode extends GameMode {
 
     private rebuildDataColumns(filters: DesignBrowserFilter[] | null = null): void {
         const FONT = Fonts.STDFONT;
-        const FONT_SIZE = 15;
+        const FONT_SIZE = UITheme.designBrowser.fontSize;
```
Comment (everyday847) [inline src/eterna/mode/DesignBrowser/DesignBrowserMode.ts:823]: good

File: src/eterna/mode/GameMode.ts [hunk 3]
```diff
@@ -20,8 +20,6 @@ import EternaURL from 'eterna/net/EternaURL';
 import Folder from 'eterna/folding/Folder';
 import Dialog from 'eterna/ui/Dialog';
 
-import NuPACK from 'eterna/folding/NuPACK';
-
```
Comment (everyday847) [inline src/eterna/mode/GameMode.ts:24]: presumably this is back from when I was toggling things based on `this._folder.name === NUpACK.NAME` or something, so... good to kill this

File: assets/Styles/styles.css (hunk 1 above)
Comment (luxaritas) [reply assets/Styles/styles.css:58]: The issue here is that there's no way to access pseudo-elements in the DOM. Looks like there are some possible workaround I wasn't aware of though

Comment (luxaritas) [reply assets/Styles/styles.css:58]: See #436

**Pull request merged**